tests/guid.test/runit

```diff
@@ -91,5 +91,65 @@ mult=`grep "^count" sel2.out | grep -v "=1" | wc -l`
 echo "found $mult with duplicate key for randomblob()"
 
 
+echo "Testing having guid as autoassigned colun via dbstore"
+
+cdb2sql ${CDB2_OPTIONS} $dbnm default 'create table t2  { 
+schema 
+{
+    byte        auto_guid[17] dbstore="GUID_FUNC"
+    int         i
+} ' &> create1.txt
+
+
+
+cdb2sql ${CDB2_OPTIONS} $dbnm default 'create table t2  { 
+schema 
+{
+    byte        auto_guid[16] dbstore="GUID"
```

Comment:
Shouldn't this be a keyword? IMO, we should differentiate between a literal and a keyword used in this context.

`dbstore = "GUID"` should store literal "GUID" .
`dbstore = GUID` should generate one.

This should also apply to CURRENT_TIMESTAMP, I think.

Reply:
Currently that gives error because csc2 format needs quotes. We can fix that later I suppose.